src/js/entry.js

```diff
@@ -68,11 +73,36 @@ class HMIApp extends React.Component {
                         <label>PTU using in-vehicle modem</label>
                     </div>
                 </div>
+                {
+                    this.props.webEngineApps.map((app) => {
+                        let query = `?sdl-host=${flags.CoreHost}&sdl-port=${flags.CoreWebEngineAppPort}&sdl-transport-role=${app.transportType.toLowerCase()}-server`;
+                        return (<WebEngineAppContainer key={app.policyAppID} policyAppID={app.policyAppID} iframeUrl={app.baseUrl + query} />);
+                    })
+                }
             </div>
         )
     }
     componentDidMount() {
         this.sdl.connectToSDL()
+
+        fileSystemController.connect(flags.FileSystemApiUrl).then(() => {
```

Comment:
please change to FileSystemController

Reply:
Find all and replace please

Reply:
fixed in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/9420e8cd33100df3cba2420ebea12091814adcac